src/appengine/libs/issue_management/issue_filer.py

```diff
@@ -257,6 +259,26 @@ def _get_from_metadata(testcase, name):
       remove_empty=True)
 
 
+def notify_issue_update(testcase, status):
+  """Notify that an issue update occurred (i.e. issue was filed or closed)."""
+  topic = local_config.ProjectConfig().get('issue_updates.pubsub_topic')
+  if not topic:
+    return
+
+  pubsub_client = pubsub.PubSubClient()
+  pubsub_client.publish(
+      topic, [
+          pubsub.Message(
+              attributes={
+                  'status': status,
+                  'testcase_id': str(testcase.key.id()),
+                  'crash_type': testcase.crash_type,
+                  'crash_state': testcase.crash_state,
```

Comment:
Add crash_address and security flag as well (i think this matches our crash params list).

Reply:
Maybe add testcase download url as well (i think they asked for reproducer as well).

Reply:
Added crash_address and security flag.

Download URL can be easily constructed by the consumer of this feed by concating domain + /download?testcase_id=ID. We wouldn't be adding much value by including this? 

In any case I don't think this feed can provide all the needed details. People will inevitably need to go to the detailed report to get all the details necessary to triage and fix the bug. This can just serve as a notification system for that.

Reply:
Friendly ping @inferno-chromium  :) Any thoughts here?

Reply:
Yes no more changes needed, and fine to skip download url.